src/components/common/TreeTag.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import { makeStyles } from 'models/makeStyles';
+
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: theme.spacing(4),
+    color: theme.palette.common.white,
+    background: theme.palette.primary.main,
+    borderRadius: theme.spacing(4),
+    width: 'fit-content',
+    letterSpacing: '0.02em',
+  },
+}));
+
+function TreeTag({ TreeTag, title, icon }) {
```

Comment:
rename TreeTag prop to fix lint warning:
`'TreeTag' is already declared in the upper scope on line 21 column 10.eslint (no-shadow)`

Reply:
Renamed the TreeTag prop to TreeTagValue.
Added a `nowrap` property to the CSS.
The font will be fixed once the branch is merged, as this branch was created before #301. That will fix this issue.

![Screenshot from 2021-12-08 11-27-15](https://user-images.githubusercontent.com/31432331/145192596-78227e91-bb81-49f9-8772-29243747e88b.png)